frontend/packages/console-plugin-sdk/src/typings/details.ts

```diff
@@ -0,0 +1,23 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { PageComponentProps } from '@console/internal/components/utils/horizontal-nav';
+import { Extension } from './extension';
+
+namespace ExtensionProperties {
+  export interface DetailViewTab {
+    model: K8sKind;
+    flag?: string;
+    href: string;
```

Comment:
What about path support?

Reply:
This can be done by reusing the `Page` type, see my comment below.